website/docs/layouts/app-frame/partials/version-history/4.10.0.md

```diff
@@ -3,9 +3,9 @@
 ### Updated
 
 `AppFrame`
-- Modified sticky/fixed position to turn off when viewport height is under 480px in height
-- Refactored styles to make `AppFrame` responsible for sticky/fixed layout of `SideNav` and `AppHeader`
+- Modified sticky/fixed position to turn off when viewport height is under 480px in height.
+- Refactored styles to make `AppFrame` responsible for sticky/fixed layout of `SideNav`.
```

Comment:
I removed references to the AppHeader in an effort to avoid confusion but let me know if you disagree and think they should be kept.